invenio_rdm_records/services/communities/service.py

```diff
@@ -149,6 +149,11 @@ def add(self, identity, id_, data, uow):
         record = self.record_cls.pid.resolve(id_)
         self.require_permission(identity, "add_community", record=record)
 
+        # Run components
+        for component in self.components:
+            if hasattr(component, "add"):
```

Comment:
I would align it with the permission so that is more readable in the instance's custom component you implement
```suggestion
            if hasattr(component, "add_community"):
```
What do you think?

Reply:
discussion: Also, should we add this hook before the `community.add(...)` call or after @slint ?